mffpy/tests/test_xml_files.py

```diff
@@ -397,6 +397,59 @@ def test_sort_categories_by_starttime(categories, idx, expected):
     assert categories.sort_categories_by_starttime()[idx] == expected
 
 
+def test_Categories_to_xml():
+    """Test `Categories.content` works with `dict2xml`"""
+    # convert some test content into an .xml of type Categories
+    expected_categories = {
+        'first category': [
+            {
+                'status': 'bad',
+                'faults': ['eyeb'],
+                'beginTime': 0,
+                'endTime': 1200000,
+                'evtBegin': 205135,
+                'evtEnd': 310153,
+                'channelStatus': {
+                    'signalBin': 1,
+                    'exclusion': 'badChannels',
+                    'channels': [1, 12, 25, 55]
+                },
+                'keys': {
+                    '#seg': {
+                        'type': 'long',
+                        'data': 3
+                    },
+                    'subj': {
+                        'type': 'person',
+                        'data': 'RM271_noise_test'
+                    }
+                }
+            }
+        ],
+    }
```

Comment:
This is the (clumsy) input to writing a "categories.xml".  My problem is that I don't fully understand what's included here, and which of the keys and items are unique.  Please check this input.